ttcc/types.py

```diff
@@ -18,4 +18,4 @@ def parse():
 
 class Number(TypeBaseClass):
     def parse():
-        pass
+        pass
```

Comment:
Leave one empty line at the end of the file or it'll keep doing ^ this! The last line shouldn't have anything, not even a space.